crud/src/main/java/com/redhat/lightblue/mediator/Mediator.java

```diff
@@ -474,7 +511,13 @@ private QueryExpression rewriteUpdateQueryForCompositeSearch(CompositeMetadata m
      * The implementation passes the request to the back-end.
      */
     @StopWatch(loggerName = "stopwatch.com.redhat.lightblue.mediator.Mediator", sizeCalculatorClass = "com.redhat.lightblue.mediator.ResponsePayloadSizeCalculator")
-    public Response find(FindRequest req) {
+    public Response find(FindRequest req, RequestMetrics metrics) {
```

Comment:
Some requests like lock, generate and streaming entity are tracked at rest layer. Lock and generate are not in mediator, so rest was the only layer they can be tracked in.Also since streaming response is handled at rest layer, monitoring streaming requests at that layer is more accurate +  it's easier to differentiate if a request is streaming or not at rest level and monitor is accordingly.